.spelling

```diff
@@ -0,0 +1,303 @@
+stateful
+readonly
+ios
+osx
+winrt
+winphone
+dotnet
+gcm
+uri
+subclassed
+v1.2.13
+v1.4.0
+v1.2.7
+e.g.
+100k
+5k
+targetable
+programmatically
+anded
+misconfiguration
+mysql
+datasets
+bbq
+subqueries
+_
+readme.md
+ultra-customizable
+signup
+recyclerview.adapter
+timestamps
+autoload
+bcrypt
+plaintext
+todo
+setdefaultacl
+defaultacl
+3.x
+4.x
+4.x.
+gradle
+x.x.x
+v3.0
+okhttp
+iot
+arduino
+repo
+unarchive
+yún
+v1.6.0
+performant
+json-encodable
+realtime
+httprequest
+npm
+http
+3xx
+url
+charset
+webhooks
+webhook
+express.js
+github
+cloudcodewebhooks-express
+_user
+_installation
+params
+enqueue
+xamarin
+validator
+async
+_jobstatus's
+heroku
+_jobstatus
+request.object
+refetched
+resaved
+datadog
+ws_connect
+ws_disconnect
+ws_disconnect_error
+websocket
+timestamp
+openssl
+json-to-csv
+mongodb
+searchable
+pre-process
+geo-queries
+geo-point
+decompile
+non-https
+friendof___
+javascript
+restapikey
+asp.net
+intellisense
+runtimes
+unencrypted
+ios7
+a.k.a.
+args
+wmappmanifest.xml.
+app.xaml.cs
+4xx
+5xx
+graphql
+sortability
+unindexed
+graphql.js
+françois
+amongst
+keychain
+xcode
+podfile
+xcworkspace
+cartfile
+subproject
+appdelegate.swift
+appdelegate.m
+subscripting
+throwable
+subclassses
+back4app
+tvos
+backgrounded
+uiapplicationdelegate
+v1.7.5
+misconfigured
+hoc
+uikit
+subspecs
+customizable
+bitwise
+datasource
+customizability
+placeholder.jpg
+localizable
+andrew
+facebookappid
+plist
+v4.0
+nodejs
+base64-encoded
+pre-compiled
+npmcdn
+node.js
+backbone.js
+websockets_api
+hostname
+classname_objectid
+classname
+codebase
+v2.3.0
+destructured
+object.set
+commonjs
+rethrow
+promisified
+apns
+watchos
+сount
+databrowser
+_linkwith
+un-authenticated
+rocksdb
+embeddable
+50x
+toolset
+i2
+r3
+ssds
+i2.xlarge
+i2.2xlarge
+i2.4xlarge
+mdadm
+pre-packaged
+percona
+mongodb.conf
+ubuntu
+cron
+mydata
+db1
+mydb
+mongodump
+mongorestore
+rs.add
+oplog
+resize
+snapshotting
+redis
+redis.createclient
+postgres
+postgis
+mlab.com
+middleware
+toolbelt
+mongolab_uri
+dev
+index.js
+analytics.js
+auth.js
+batch.js
+cache.js
+classes.js
+config.js
+crypto.js
+databaseadapter.js
+exportadapter.js
+facebook.js
+files.js
+filesadapter.js
+fileloggeradapter.js
+functions.js
+gridstoreadapter.js
+installations.js
+loggeradapter.js
+middlewares.js
+promiserouter.js
+push.js
+rest.js
+restquery.js
+restwrite.js
+roles.js
+schema.js
+sessions.js
+transform.js
+triggers.js
+users.js
+aes256-gcm
+s3_access_key
+s3_secret_key
+s3_bucket
+s3adapter
+gcsadapter
+fsadapter
+parse-server-fs-adapter
+node.js.
+postgresql
+2.x
+whitelist
+runtime
+dynos
+json
+androidmanifest.xml
+macos
+raywenderlich.com
+appcoda.com
+janrain
+keycloak
+meetup
+phantauth
+spotify
+vkontakte
+weibo
+oauth
+userinfo
+forumsys
+nginx.conf
+composer.json
+autoloader
+__construct
+geo
+parsable
+superglobal
+postfixed
+that'll
+geolocations
+plott's
+lt
+lte
+gte
+nin
+dontselect
+tokenizer
+unaccent
+2xx
+wi-fi
+1.0a
+unlinking
+nuget-link
+5.2.x
```

Comment:
I assume this file is the "ignore" list for the spell check. There are quite a lot of version numbers, does this accept a regex to cover all these version numbers?

Reply:
I don't believe so

Reply:
Could you sort the contents of this file alphabetically? Just for better overview.